src/AppLayout.js

```diff
@@ -10,19 +10,19 @@ const TABS = [
     key: 'tabs.scenario.key',
     label: 'layouts.tabs.scenario.tab.title',
     to: '/scenario',
-    render: () => <Scenario /> // eslint-disable-line
+    render: <Scenario /> // eslint-disable-line
```

Comment:
I don't remember why we disabled eslint on this line, could you check if it is still useful after your change ? Thank you ! :smile:

Reply:
I've already tried, we still need it because "React' must be in scope when using JSX" :slightly_smiling_face:

Reply:
Ok, thank you :)